config/i18n-tasks.yml

```diff
@@ -89,6 +89,11 @@ ignore_unused:
   - 'activerecord.*'
   - 'attributes.*'
   - 'diagnoses.steps.*'
+  - 'mailers.user_mailer.yesterday_modifications.waiting_for_answer'
```

Comment:
Why not only `mailers.user_mailer.yesterday_modifications.*` instead of those 5 lines?

Reply:
Why not be specific ? Those are the only one needing an exception

Reply:
There are no other keys under yesterday_modifications.* level